tests/azul_agent.py

```diff
@@ -34,7 +33,7 @@ def get_files_py_project(self, project_shortname):
         # https://github.com/DataBiosphere/azul/issues/537
         params = dict(filters=str(filters), size=str(size))
         while True:
-            url = self.azul_url + '/repository/files?' + urllib.parse.urlencode(params, safe="{}/'")
+            url = self.azul_url + '/repository/' + entity_type + '?' + urllib.parse.urlencode(params, safe="{}/'")
```

Comment:
Just a suggestion, but if you'd like to avoid extra operators:
```
url = self.azul_url + f"/repository/{entity_type}?" + urllib.parse.urlencode(params, safe="{}/'")
```

Reply:
@xbrianh  proposal mixes f"" strings and string concatenation. Either do one or the other, @jessebrennan. But note that `f""` silently applies `str()` to every interpolated value. Concatenation doesn't. This is a significant semantic difference.